### PR TITLE
ignore_ssl on http executors

### PR DESCRIPTION
**Pull request by bewiwi**

### CODE REVIEW

File: executors/http/README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ In your yaml file, you can use:
   - body optional
   - bodyFile optional
   - headers optional
+  - ignore_ssl optional
```
Comment (yesnault) [inline executors/http/README.md]: ignore_verify_ssl optional: set to true if you use a self-signed SSL on remote for example

File: executors/http/executor.go [hunk 2]
```diff
@@ -39,6 +40,7 @@ type Executor struct {
 	BodyFile      string      `json:"bodyfile" yaml:"bodyfile"`
 	MultipartForm interface{} `json:"multipart_form" yaml:"multipart_form"`
 	Headers       Headers     `json:"headers" yaml:"headers"`
+	IgnoreSSL     bool        `json:"ignore_ssl" yaml:"ignore_ssl"`
```
Comment (yesnault) [inline executors/http/executor.go]: IgnoreVerifySSL     bool        `json:"ignore_verify_ssl" yaml:"ignore_verify_ssl"`

File: executors/http/executor.go [hunk 3]
```diff
@@ -82,8 +84,13 @@ func (Executor) Run(testCaseContext venom.TestCaseContext, l venom.Logger, step
 		req.Header.Set(k, v)
 	}
 
+    tr := &http.Transport{
+        TLSClientConfig: &tls.Config{InsecureSkipVerify: t.IgnoreSSL},
```
Comment (yesnault) [inline executors/http/executor.go]: TLSClientConfig: &tls.Config{InsecureSkipVerify: t.IgnoreVerifySSL},

**Comment by yesnault** [comment]
this will close #97

**Comment by bewiwi** [comment]
Don't merge now, seems it's not working

**Pull request merged**